llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -1615,19 +1615,39 @@ static bool areIdenticalUpToCommutativity(const Instruction *I1,
   return false;
 }
 
+static bool isSafeCheapLoadStore(const Instruction *I,
```

Comment:
Move this as an NFC to simplify the patch

Reply:
https://github.com/llvm/llvm-project/pull/109398